docs/install-github.md

```diff
@@ -40,69 +40,22 @@
    2.2. Save the JFrog connection details as a [JFrog Platform Access Token Integration](https://www.jfrog.com/confluence/display/JFROG/JFrog+Platform+Access+Token+Integration)
    named **jfrogPlatform**.
 
-   2.3. Save your GitHub access token as a [Generic Integration](https://www.jfrog.com/confluence/display/JFROG/Generic+Integration) named **github** with the token as the key and the GitHub access token as the value.
+   2.3.Save your Github access token in a [GitHub Integration](https://www.jfrog.com/confluence/display/JFROG/GitHub+Enterprise+Integration) named
+   **gitIntegration**.
 
    2.4. Set the `.jfrog-pipelines` directory in the root of your **Frogbot Management Repository**.
```

Comment:
What is the purpose of this section? Why should the user create this directory?

Reply:
It is mandatory dir for Pipelines